developer-docs/account/ENVIRONMENT.md

```diff
@@ -10,7 +10,7 @@ This application recognizes the following environment variables:
 | `CACHE_KEY_PREFIX`                        | Prefix to use for Redis cache keys                                                                                                                                                |                                                  string                                                  |     Y     |                                                 |
 | `CAPACITY_LIMIT`                          | Maximum amount of provider capacity this app is allowed to use (per epoch) type: 'percentage' 'amount' value: number (may be percentage, ie '80', or absolute amount of capacity) | JSON [(example)](https://github.com/ProjectLibertyLabs/gateway/blob/main/env-files/account.template.env) |     Y     |                                                 |
 | `FREQUENCY_HTTP_URL`                      | Blockchain node address resolvable from the client browser                                                                                                                        |                                               http(s): URL                                               |     Y     |                                                 |
-| `FREQUENCY_URL`                           | Blockchain node address                                                                                                                                                           |                                          http(s): or ws(s): URL                                          |     Y     |                                                 |
+| `FREQUENCY_WS_URL`                        | Blockchain node address                                                                                                                                                           |                                          http(s): or ws(s): URL                                          |     Y     |                                                 |
```

Comment:
Is this comment correct? We never reference it as an http, only a ws.

Reply:
I think it should only be `WS` if we are renaming the config to reflect that.